src/app/cart/cart.component.js

```diff
@@ -60,8 +60,17 @@ class CartController {
     this.$window.location.href = this.sessionService.getRole() === 'REGISTERED' ? 'checkout.html' : 'sign-in.html';
   }
 
-  donationStartDate( month, day ) {
-    return new Date( 0, Number( month ) - 1, Number( day ) );
+  donationStartDate( config ) {
```

Comment:
So when you load the cart, do you have all the data you need? You just take the configured day from the EP cart and figure out the next time that day occurs on or after the next Siebel draw day right? I think this could be good as part of the cart service as it could be used throughout checkout. Although I could see this date getting off if they go through checkout slowly and the Siebel draw day changes.